scalafix-nsc/src/test/resources/checkSyntax/OrganizeImportsNoGroups.source

```diff
@@ -0,0 +1,28 @@
+rewrites = []
+imports.organize = true
+imports.alwaysUsed = []
```

Comment:
Can we remove unnecessary lines from this configuration to highlight that `imports.groups = []` is the line being tested?